src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
+  --color: #00acdc;
+
+  font-family: Roboto, serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  padding-top: 22px;
+  padding-bottom: 22px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.nav__item {
+  font-weight: 500;
```

Comment:
The font-weight of 500 might not be supported by all fonts. Ensure that the 'Roboto' font includes this weight or consider using a more common weight like 400 or 600.